index.html

```diff
@@ -1688,5 +1688,57 @@ <h2>MediaStream in workers</h2>
 };</pre>
     </div>
   </section>
+  <section>
+    <h2>Background segmentation mask</h2>
+    <p>Some platforms or User Agents may provide built-in support for background segmentation of video frames, in particular for camera video streams.
+       Web applications may want to control whether background segmentation is computed at the source level and to get access to the computed segmentation masks.
+       This allows the web application for instance
+       to do custom framing or background blurring or replacement
+       while leveraging on platform computed background segmentation.
+       This allows the web application
+       to access the original unmodified frame and
+       to fine tune frame modifications based on its likings.
+       For that reason, we extend {{MediaStreamTrack}} with the following properties and {{VideoFrame}} with the following attributes.
+    </p>
+    <pre class="idl">
+partial dictionary MediaTrackSupportedConstraints {
+  boolean backgroundSegmentationMask = true;
+};
+
+partial dictionary MediaTrackConstraintSet {
+  ConstrainBoolean backgroundSegmentationMask;
```

Comment:
Would it ever be interesting and feasible to tweak the parameters by which segmentation is done?

Reply:
Atleast on Windows, the platform model does not allow tweaking segmentation parameters **today**. Using [tensorflow.js](https://github.com/tensorflow/tfjs-models/tree/master/body-segmentation#example-code-and-demos) with `BodyPix` model for Blur, I see there's atleast a `segmentationThreshold` parameter. Maybe it's the same as `foregroundThresholdProbability `  with the `MediaPipeSelfieSegmentation ` model ?  
Did you have any more parameters in mind ?

![mediapipe_parameters](https://github.com/w3c/mediacapture-extensions/assets/975872/26158266-b1d2-48c0-9490-5eea734a5e0e)

Reply:
> Did you have some other parameters in mind?

I am not knowledgeable enough on what parameters would be best to include. I was mostly wondering if this is something we foresee extending from a boolean to a set of parameters, and if so, whether there was a viable path for such future extensions given the current API shape.

Reply:
In Media Capture API, the parameter space is flat and not hierarchical.

As an example, there is a constrainable property called `whiteBalanceMode` which can be constrained to `manual`. If one then wants to manually change the white balance, there is a constrainable property called `colorTemperature` which can be constrained separately in order to do that.

So if we later would like to add a numeric constrainable property called `backgroundSegmentationThreshold` (which could change the segmentation mask to be pre-processed to an blank and white mask according to the threshold without shades of grey) or a string constrainable property called `backgroundSegmentationModel` (to use the particular AI model), we could certainly do that.